common/src/main/java/org/broadleafcommerce/common/email/service/info/EmailInfo.java

```diff
@@ -152,7 +152,7 @@ public void setEncoding(String encoding) {
 
     public synchronized EmailInfo clone() {
         EmailInfo info = new EmailInfo();
-        info.setAttachments(attachments);
+        info.setAttachments(new ArrayList<Attachment>());
```

Comment:
Let's create a new ArrayList and copy the attachments.  If the "client" needs to alter what is in the attachments, that would be logic they would need to implement.  A clone should be a full copy.